packages/embark-typings/babel.config.js

```diff
@@ -0,0 +1,49 @@
+/* global module */
+
+module.exports = function (api) {
+  const node = {
+    ignore: [
```

Comment:
these ignores are relevant only to `packages/embark`.

Reply:
Actually, since all the stuff in typings is `d.ts` files, which are used by `tsc` to typecheck (maybe tslint looks at them too), babel may not be needed at all and there may not need to be a build script in this package.